feat(rum): disable beacon compression when flag is set (#1214)
elastic/apm-agent-rum-js: docs/troubleshooting.asciidoc

3. Wait at least 5 seconds after the page has loaded
4. Monitor the Console and Network panel in your browsers developer tools

[float]
[[disable-events-payload-compression]]
=== Disable events payload compression

> In browsers such as Chrome the payload of the events sent by the RUM JS agent is being compressed with gzip.
> Because of that, you will not see a readable content when inspecting the event with the Network panel of your browser developer tools.

Reviewer: ```suggestion
In browsers such as Chrome, the RUM agent event payload is compressed with gzip.
Because of this compression, you will not see readable content when inspecting the event with the Network panel of your browser developer tools.
```

There are situations where that can be an issue. For instance, HAR files will not show a readable information. Therefore, inspecting events
for debugging purposes will not be possible.

There are two ways to disable the payload compression:

1. Creating an item named `_elastic_inspect_beacon_` using the sessionStorage browser API.

> 2. Loading the webpage including the query param `_elastic_inspect_beacon_` in the URL. E.g. `https://elastic.co?_elastic_inspect_beacon_`

Reviewer 2: https://elastic.co?_elastic_inspect_beacon_=true
